src/test/java/nextstep/subway/acceptance/PathAcceptanceTest.java

```diff
@@ -0,0 +1,135 @@
+     *          |                        |
+     *        남부터미널역  --- *3호선* ---   양재
+     *                        5
+     */
+
+    @BeforeEach
+    void set() {
+        교대역 = 지하철역_생성_응답("교대역");
+        강남역 = 지하철역_생성_응답("강남역");
+        양재역 = 지하철역_생성_응답("양재역");
+        남부터미널역 = 지하철역_생성_응답("남부터미널역");
+
+        신분당선 = 지하철_노선_생성("신분당선", "bg-red-600", 강남역.getId(), 양재역.getId(), DEFAULT_DISTANCE)
+                .jsonPath()
+                .getObject("", LineResponse.class);
+        이호선 = 지하철_노선_생성("2호선", "bg-green-600", 강남역.getId(), 교대역.getId(), DEFAULT_DISTANCE)
+                .jsonPath()
+                .getObject("", LineResponse.class);
+        삼호선 = 지하철_노선_생성("3호선", "bg-orange-600", 교대역.getId(), 남부터미널역.getId(), DEFAULT_DISTANCE)
+                .jsonPath()
+                .getObject("", LineResponse.class);
+
+        지하철_노선_구간_등록(삼호선.getId(),남부터미널역.getId(),양재역.getId(),SHORT_DISTANCE);
+
+
+    }
+
+    /**
+     * When 출발역과 도착역으로 경로를 조회하면
+     * Then 출발역으로부터 도착역까지의 경로에 있는 역 목록과 조회한 경로 구간의 거리를 응답받는다.
+     */
+    @DisplayName("최단 경로 조회")
+    @Test
+    void getPaths() {
+        Long source = 교대역.getId();
+        Long target = 양재역.getId();
+
+        ExtractableResponse<Response> response = 최단_경로_조회(BASE_URL + "?source=" + source + "&target=" + target);
+
+        PathResponse path = response.jsonPath().getObject("", PathResponse.class);
+
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+
+        int stationCount = path.getStations().size();
+        assertThat(stationCount).isEqualTo(3);
+        assertThat(path.getStations().stream().mapToLong(it->it.getId())).containsExactly(교대역.getId(),남부터미널역.getId(),양재역.getId());
+        assertThat(path.getDistance()).isEqualTo(DEFAULT_DISTANCE+SHORT_DISTANCE);
+    }
+
+    /**
+     * When 출발역과 도착역을 동일하게 입력 후 경로를 조회하면
+     * Then 에러를 던진다.
+     */
+    @DisplayName("최단 경로 조회 - 출발역과 도착역이 같은 경우 에러를 던진다.")
+    @Test
+    void getPathsExceptionWhenSameSourceAndTarget() {
+        Long stationId = 교대역.getId();
+
+        ExtractableResponse<Response> response = 최단_경로_조회(BASE_URL + "?source=" + stationId + "&target=" + stationId);
+
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.BAD_REQUEST.value());
+    }
+
+    /**
+     * When 출발역과 도착역이 연결이 되어 있지 않은 경로를 조회하면
+     * Then 에러를 던진다.
+     */
+    @DisplayName("최단 경로 조회 - 출발역과 도착역이 연결이 되어 있지 않은 경우 에러를 던진다.")
+    @Test
+    void getPathsExceptionWhenNotConnectedPath() {
+        //given
+        Long 신도림역_아이디 = 지하철역_생성_응답("신도림역").getId();
+        Long 용산역_아이디 = 지하철역_생성_응답("용산역").getId();
+        LineResponse 일호선 = 지하철_노선_생성("1호선", "bg-blue-600", 신도림역_아이디, 용산역_아이디, DEFAULT_DISTANCE)
+                .jsonPath()
+                .getObject("", LineResponse.class);
+
+        //when
+        Long source = 교대역.getId();
+        Long target = 신도림역_아이디;
+        ExtractableResponse<Response> response = 최단_경로_조회(BASE_URL + "?source=" + source + "&target=" + target);
```

Comment:
아래처럼 호출하도록 하면 더 가독성 있을 것 같아요. 😀
```suggestion
        ExtractableResponse<Response> response = 최단_경로_조회(source, target);
```